jjb/jobs/qpc-nightly-automation.yaml

```diff
@@ -3,6 +3,11 @@
     node: 'f27-os'
     triggers:
         - timed: '@midnight'
+    properties:
+      - build-blocker:
+        blocking-jobs:
+          - 'qpc-.*nightly-automation'
+        queue-scanning: 'BUILDABLE'
```

Comment:
Here the intention is to block until a PILOT job is running. Maybe we could use buildable for both since the trigger will throttle the jobs.

But I am looking for creating a CD/CI pipeline and I will be making this job run whenever the corresponding docker image build job completes successfully.